docs/source/flow_setup.rst

```diff
@@ -228,10 +246,48 @@ jobs from there.
     ray teardown scripts/ray_autoscale.yaml
 
 
-Custom configuration
+9. Custom configuration
 ====================
 
 You may define user-specific config parameters as follows
 ::
 
     cp flow/core/config.template.py flow/core/config.py  # Create template for users using pycharm
+
+
+Remote desktop using Docker for simple utilisation of flow (recommended for testing purpose)
+******************
+
+1. Installation
+====================
+
+Installation of a remot desktop and docker to get access to flow quickly
+
+First install docker on https://www.docker.com/
+
+In terminal
+::
+    1° docker pull lucasfischerberkeley/flow_desktop
+    2° docker run -d -p 5901:5901 -p 6901:6901 -p 8888:8888 lucasfischerberkeley/flow_desktop
+    
+Go into your browser ( Firefox, Chrome, Safari)
+::
+    1° Go to http://localhost:6901/?password=vncpassword
+    2° Go to Applications and open Terminal Emulator
+    3° For sumo: Write python flow/examples/sumo/sugiyama.py and run it
+    4° For rllib : Write python flow/examples/rllib/stabilizing_the_ring.py and run it
```

Comment:
so rllib is not installed in a virtual env but rllab is?

Reply:
@eugenevinitsky. Rllib is installed in the root environment and works.
For rllab, I need to enter another environment ( flow-rllab) but I am working on a way to have everything set up in the same environment

Reply:
Okay, should be good to go after this. Can you add the dockerfile that you used to create the docker image? Would be useful to have in the docker folder as a record if we need to remake it!